### ISSUE TITLE
In new Media Manager not an option to "empty" image?

### ISSUE THREAD

**Issue by KiraTheUltimate** [e107inc/e107#3392]
Earlier had an option to set up "empty" image, if I don't want to use image... but now, there's no option, so if I set up an image I can't delete it. (version: git)

Now:

![01](https://user-images.githubusercontent.com/12445784/44312323-83018f80-a3f6-11e8-9e7b-3b4215608a66.JPG)

Earlier:

![02](https://user-images.githubusercontent.com/12445784/44312325-8c8af780-a3f6-11e8-8187-10e39a92bd19.JPG)

**Comment by CaMer0n** [comment]
@KiraTheUltimate Yes, this is on the TODO list. :+1:  Thank you for the reminder.
